message.js

```diff
@@ -228,7 +236,7 @@ jspb.Message.SUPPORTS_UINT8ARRAY_ = (typeof Uint8Array == 'function');
 /**
  * The internal data array.
  * @type {!Array}
- * @protected
+ * @export
```

Comment:
i don't think this is accessed by any of the gencode, is it? we should be able to just change this to private with no need to export

Reply:
rolled back to `@protected`